packages/compass-e2e-tests/index.ts

```diff
@@ -98,6 +98,10 @@ function cleanup() {
       debug('Failed to clean up server data', e);
     }
   }
+
+  // Since the webdriverio update something is messing with the terminal's
+  // cursor. This brings it back.
+  crossSpawn.sync('tput', ['cnorm'], { stdio: 'inherit' });
```

Comment:
I don't know but without this line the tests are maddening to work on locally ;)